source/MRViewer/MRSurfaceManipulationWidget.h

```diff
@@ -155,6 +160,8 @@ class MRVIEWER_CLASS SurfaceManipulationWidget :
 
     std::shared_ptr<Palette> palette_;
     bool enableDeviationTexture_ = true;
+    bool deviationAsShift_ = false;
```

Comment:
Let as make three modes:
1) point-to-point (now named by-shift)
2) point-to-plane (new)
3) exact-distance

Reply:
updated